### PR TITLE
WS-03: Created empty repos

### PR DESCRIPTION
**Pull request by NickOsman11**

### CODE REVIEW

File: backend/Repositories/LocationRepo.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using System.Linq;
```
Comment (TimLeach635) [inline backend/Repositories/LocationRepo.cs:1]: You're not using Linq in this file! Remove

File: backend/Repositories/LocationRepo.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using WhaleSpotting.Models.Database;
```
Comment (TimLeach635) [inline backend/Repositories/LocationRepo.cs:2]: I also believe you're not using this namespace?

File: backend/Repositories/SightingRepo.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+using System.Linq;
+using WhaleSpotting.Models.Database;
```
Comment (TimLeach635) [inline backend/Repositories/SightingRepo.cs:2]: Ditto

File: backend/Repositories/SightingRepo.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+    public class SightingRepo : ISightingRepo
+    {
+        private readonly WhaleSpottingDbContext _context;
+
+        public SightingRepo(WhaleSpottingDbContext context)
+        {
+            _context = context;
+        }
+
```
Comment (TimLeach635) [inline backend/Repositories/SightingRepo.cs:18]: Rogue empty line

File: backend/Repositories/WhaleRepo.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+using System.Linq;
+using WhaleSpotting.Models.Database;
```
Comment (TimLeach635) [inline backend/Repositories/WhaleRepo.cs:2]: Ditto

File: backend/Repositories/WhaleRepo.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+    public class WhaleRepo : IWhaleRepo
+    {
+        private readonly WhaleSpottingDbContext _context;
+
+        public WhaleRepo(WhaleSpottingDbContext context)
+        {
+            _context = context;
+        }
+
```
Comment (TimLeach635) [inline backend/Repositories/WhaleRepo.cs:18]: Ditto

File: backend/Startup.cs [hunk 7]
```diff
@@ -57,9 +57,13 @@ public void ConfigureServices(IServiceCollection services)
       services.AddDbContext<WhaleSpottingDbContext>();
 
       services.AddTransient<IUserRepo, UserRepo>();
+      services.AddTransient<ILocationRepo, LocationRepo>();
+      services.AddTransient<ISightingRepo, SightingRepo>();
+      services.AddTransient<IWhaleRepo, WhaleRepo>();
 
       services.AddTransient<IAuthService, AuthService>();
       services.AddTransient<IUserService, UserService>();
+
```
Comment (TimLeach635) [inline backend/Startup.cs:66]: Extra blank line!

File: backend/Startup.cs [hunk 8]
```diff
@@ -57,9 +57,13 @@ public void ConfigureServices(IServiceCollection services)
       services.AddDbContext<WhaleSpottingDbContext>();
 
       services.AddTransient<IUserRepo, UserRepo>();
+      services.AddTransient<ILocationRepo, LocationRepo>();
+      services.AddTransient<ISightingRepo, SightingRepo>();
+      services.AddTransient<IWhaleRepo, WhaleRepo>();
```
Comment (TimLeach635) [inline backend/Startup.cs:62]: Well remembered 😉

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Changes for you!

File: backend/Startup.cs (hunk 8 above)
Comment (NickOsman11) [reply backend/Startup.cs:62]: Thank you!

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**